scss/detail.scss

```diff
@@ -0,0 +1,67 @@
+* {
+  font-family: $font-family;
+}
+
+body {
+  background-color: #F4F4F4;
```

Comment:
Use variables...
I think the best way is create a new file and import this variables. ☺️